### ISSUE TITLE
procinfo should display cwd and cmdline as well

### ISSUE THREAD

**Issue by disconnect3d** [pwndbg/pwndbg#1876]
Heh:
```
pwndbg> info proc
process 2005
cmdline = '/opt/teamviewer/tv_bin/teamviewerd -d'
cwd = '/opt/teamviewer'
exe = '/opt/teamviewer/tv_bin/teamviewerd'
pwndbg> procinfo
exe        '/opt/teamviewer/tv_bin/teamviewerd'
pid        2005
tid        2005
ppid       1
uid        [0, 0, 0, 0]
gid        [0, 0, 0, 0]
groups     []
fd[0]      /dev/null
fd[1]      /dev/null
fd[2]      /dev/null
fd[3]      anon_inode:[eventfd]
fd[4]      anon_inode:[eventpoll]
fd[5]      anon_inode:[timerfd]
fd[6]      unix '(anonymous)'
fd[7]      pipe:[56989]
fd[8]      pipe:[56989]
fd[9]      /run/teamviewerd.pid
fd[10]     /var/log/teamviewer15/TeamViewer15_Logfile.log
fd[11]     /var/log/teamviewer15/TVNetwork.log
fd[12]     tcp 127.0.0.1:5939 => 0.0.0.0:0 (listen)
fd[13]     /run/teamviewerd.ipc
fd[14]     tcp 192.168.1.152:53028 => 34.105.162.41:5938 (established)
```

**Comment by dmur1** [comment]
i will take a look at this

**Comment by dmur1** [comment]
see https://github.com/pwndbg/pwndbg/pull/1879

**Comment by disconnect3d** [comment]
Fixed in #1879
